### DISCUSSION TITLE
datasets/explorer/dashboard/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8196] [Docs]
# datasets/explorer/dashboard/

Learn about Ultralytics Explorer GUI for semantic search, SQL queries, and AI-powered natural language search in CV datasets.

https://docs.ultralytics.com/datasets/explorer/dashboard/

**Comment by vonzo** [comment]
How do I run this with my own data? and my own model?

**Reply by glenn-jocher** [reply]
To run YOLOv8 with your own data and model, you'll need to follow these steps:

1. Prepare your dataset in a format that YOLOv8 can understand, typically consisting of images and corresponding annotation files.
2. Train your model using the `yolo train` command, specifying your dataset and model configuration file.
3. After training, you can validate, make predictions, or use other features like tracking with your trained model.

Here's a quick example of how you might train your model:

```python
from ultralytics import YOLO

# Create a new YOLO model from your custom config
model = YOLO('path/to/your/model.yaml')

# Train the model on your custom dataset
model.train(data='path/to/your/dataset.yaml')
```

And to make predictions:

```python
# Load your trained model
model = YOLO('path/to/your/best.pt')

# Predict on new images
results = model.predict('path/to/your/images')
```

Remember to replace `'path/to/your/...'` with the actual paths to your files. If you need more detailed guidance, our [documentation](https://docs.ultralytics.com/) is a great place to start. 😊

**Comment by hannaliavoshka** [comment]
How can I run the Explorer GUI with my custom dataset and my own model?

**Reply by jakhon37** [reply]
https://docs.ultralytics.com/datasets/explorer/

**Reply by glenn-jocher** [reply]
@jakhon37 the Ultralytics Explorer GUI has been deprecated as of version 8.3.10. However, you can use the Ultralytics HUB for similar functionality, including data exploration and management. For more details, visit the [Ultralytics HUB](https://ultralytics.com/hub).

**Comment by ws1065** [comment]
I executed yolo Explorer to try to open Explorer dashboard, but it shows that it should be a port monitoring error? I don't know what port it is listening on, and yolo explorer doesn't have a help command?

**Reply by glenn-jocher** [reply]
@ws1065 it sounds like you're encountering a port conflict issue when trying to run the Explorer dashboard. By default, the Explorer GUI listens on port 8080. You can specify a different port by using the `--port` argument when launching the GUI. For example, you can try `yolo explorer --port 8081`. If the issue persists, please ensure you are using the latest version of the package by updating it with `pip install --upgrade ultralytics[explorer]`. For further details, you can refer to the [Ultralytics Explorer documentation](https://docs.ultralytics.com/datasets/explorer/dashboard/).
